models/categories.models.js

```diff
@@ -0,0 +1,11 @@
+const db = require("../db/connection");
+
+exports.fetchCategories = () => {
+  if (!"/api/categories") {
+    return Promise.reject({ status: 404, message: "Not Found" });
+  }
+  return db.query(`SELECT * FROM categories`).then((categories) => {
+    // console.log(categories.rows);
```

Comment:
Forgivable - but we'll be really strict with "leftover console.logs" this week, get used to removing them now 😄